bee-inx/src/ledger.rs

```diff
@@ -1,6 +1,7 @@
 // Copyright 2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
+use bee::payload::milestone::MilestoneIndex;
```

Comment:
Should be aligned with the other imports, why don't import them all at the top?

Reply:
Generally I agree with you, but here it makes it easier to see where the types come from, in my opinion.

Reply:
But why import the MilestoneIndex then also not in the code where needed?